ansible_ai_connect/ai/api/versions/v1/urls.py

```diff
@@ -24,4 +29,6 @@
     path("me/", include(me_urls)),
     path("telemetry/", include(telemetry_urls)),
     path("wca/", include(wca_urls)),
+    path("health/", WisdomServiceLivenessProbeView.as_view(), name="health"),
```

Comment:
OK, so these effectively _move_ the **unversioned** `/check` and `/check/status` [endpoints](https://github.com/ansible/ansible-ai-connect-service/blob/7f64f1991952b31fbac4e97147fab355a3ac136c/ansible_ai_connect/main/urls.py#L65-L66) to **versioned** `/v1/health` and `/v1/health/status`?

Reply:
yes we need that endpoints in the api/... leaf, so the api/v1 is the ideal location. 
I am not removing the old endpoints in the root /check and /check/status/ , to not break api compatibility that may be used globally.